tests/drivers/spi/spi_loopback/src/spi.c

```diff
@@ -430,6 +430,11 @@ static int spi_rx_every_4(const struct device *dev,
 		return 0;
 	}
 
+	if (IS_ENABLED(CONFIG_DSPI_MCUX_EDMA)) {
+		LOG_INF("Skip every 4");
```

Comment:
should `LOG_INF()` be removed prior to merge?

Reply:
ok